comments.md

```diff
@@ -0,0 +1,44 @@
+# New Comments
+-(ZH) I'm now thinking that when you are "Building" resources, you should be using only a build definition object.  Then once, it is deployed, it would return a PowerFormResource that contains your build defnition as well as the cloud definitions.  Thoughts on this?
+
```

Comment:
yeah, keep exploring that path.  i think makes sense to flesh that out in the 'implementation scenarios' or 'use cases' we build out

Reply:
> yeah, keep exploring that path. i think makes sense to flesh that out in the 'implementation scenarios' or 'use cases' we build out

yeah, I am just finding as we use "resource definition" it is becoming more and more like the "manager" issue we had.  We are now passing around a bunch of objects that are not needed during a build phase.  We wouldn't need cloud objects ever during a build phase.  That's where functions for "load" would return the cloud objects if we need scope, identity ids, etc.